open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -13,10 +13,13 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    ax=df[df.index&lt;"2020-06-01"]['ninfected'].diff().rolling(7).mean().plot()
</pre><p>you can create a function as filter_data_before(df, datetime).</p><p> </p><p>Then you can have <code>get_rolling_window(diff(filter_data_before(df, '2020-06-01')['ninfected'], 7).plot()</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>DONE</p>